test/db_test.py

```diff
@@ -5,18 +5,18 @@
 from peewee import IntegrityError
 from playhouse.shortcuts import model_to_dict
 from pocha import before, beforeEach, describe, it
-import scrypt
 
 import testutil
 from db import Code, LoginCode, PendingEmail, User
```

Comment:
Because of the last refactor, I wonder, should db not be before testutil in the imports?